renovate.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["config:base"],
```

Comment:
Just a question, where this is coming from? It appears that it's not a [renovate config](https://docs.renovatebot.com/presets-config/)

Reply:
I think I took an example from crowdsec

relevant to node @Tbaile , this PR only list collabora

```
chore(deps): update collabora/code docker tag to v24.04.5.2.1
Schedule: ["at any time"]
Branch name: renovate/collabora-code-24.x
Merge into: main
Upgrade collabora/code to 24.04.5.2.1
```

what do you think ?

Reply:
For now, applying the changes that @DavidePrincipi provided (using a <depname>) will trigger node updates too, just beware of that.

Just found out that `config:base` is an old ref for `config:recommended`, you can safely swap that since `config:base` is being translated in the [renovate application](https://developer.mend.io/github/NethServer/ns8-collabora/-/job/8a7da7ee-133b-46a3-b8af-2d4a6a39f8bb). (under `Config migration necessary`)